runtime/help/commands.md

```diff
@@ -17,13 +17,13 @@ by pressing `Ctrl-e` and entering the command. Arguments are placed in single
 quotes here but these are not necessary when entering the command in micro.
 
 * `bind 'key' 'action'`: creates a keybinding from key to action. See the
-   `keybindings` documentation for more information about binding keys.
+  `keybindings` documentation for more information about binding keys.
```

Comment:
This is not exactly what I meant... The "body" of the description below "\`bind\` ...", including those lines that due to circumstances begin with \` ("\`keybindings\` ..."  and "\`key\` ..." in this case), should all have the same consistent indentation, i.e. 3 spaces in this case.